contractcourt/briefcase.go

```diff
@@ -125,6 +125,59 @@ type ArbitratorLog interface {
 type ArbitratorState uint8
 
 const (
+	// While some state transition is allowed, certain transitions are not
+	// possible. Listed below is the full state transition map which
+	// contains all possible paths. We start at StateDefault and end at
+	// StateFullyResolved, or StateError(not listed as its a possible state
+	// in every path). The format is,
+	// 	-> state: conditions we switch to this state.
+	//
+	// StateDefault
```

Comment:
Very cool state transition diagram! Would be cool to eventually use something to render this out using a traditional state-machine like structure. This is a big improvement on what we have for now though imo.